front/app/modules/commercial/content_builder/admin/components/contentBuilderToggle.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen } from 'utils/testUtils/rtl';
+import ContentBuilderToggle from './contentBuilderToggle';
+import { WithRouterProps } from 'react-router';
+
+jest.mock('services/locale');
+jest.mock('services/avatars');
+jest.mock('utils/cl-intl');
+jest.mock('services/appConfiguration');
+jest.mock('utils/cl-router/history');
+jest.mock('components/UI/QuillEditor/QuillMultilocWithLocaleSwitcher');
+
+const getRouterProps = (projectId) =>
+  ({
+    location: {
+      pathname: '/admin/projects/projectID/description',
+    },
+    params: {
+      projectId,
+    },
+  } as any as WithRouterProps);
+
+const dummyOnChange = () => {};
+
+const routerProps = getRouterProps('projectID');
+
+describe('ContentBuilderToggle', () => {
+  it('Confirm link shown appropriately when builder option toggled', () => {
+    render(
+      <ContentBuilderToggle
+        valueMultiloc={undefined}
+        onChange={dummyOnChange}
+        label={'Label'}
+        labelTooltipText={'LabelTooltipText'}
+        toggleLabel={'ToggleLabel'}
+        toggleTooltipText={'ToggleTooltipText'}
+        linkText={'LinkText'}
+        {...routerProps}
+      />
+    );
+    const toggle = screen.getByTestId('toggle');
```

Comment:
Maybe switch go `getByRole` for the toggle and the link, if possible? If too much effort - fine to leave it as is!

Reply:
@IvaKop I changed to getByRole for the toggle. Since I'm testing that the link is not in the document, it throws an error (rather than returning null) if I use getByRole.

Is it better practice to expect that getByRole throws an error, or to use queryByText and use "not.toBeInTheDocument"?